airbyte-config/init/src/main/resources/config/STANDARD_SOURCE_DEFINITION/12928b32-bf0a-4f1e-964f-07e12e37153a.json

```diff
@@ -2,7 +2,7 @@
   "sourceDefinitionId": "12928b32-bf0a-4f1e-964f-07e12e37153a",
   "name": "Mixpanel",
   "dockerRepository": "airbyte/source-mixpanel",
-  "dockerImageTag": "0.1.7",
+  "dockerImageTag": "0.1.8.1",
```

Comment:
Is it expected to user an extra `.1` version after the patch version number?
@sherifnada?

Reply:
It was my workaround), it worked normally, as for minor fix, specifically for this connector there was a gap in the version number, because the 0.1.8 was already released with no changelog attached, so decided to bump it like this. Hope this is not the problem)

Reply:
I missed this in my review, our system assumes the format `<major>.<minor>.<patch>` in some places so we should stick to that format. If this is the latest version we should publish another one with the right format to correct.

Reply:
I'll handle this asap. Thanks for the clarification.

Reply:
Fixed in https://github.com/airbytehq/airbyte/pull/9320